cli_tools_e2e_test/gce_image_import_export/test_suites/import/cli_tests.go

```diff
@@ -372,6 +384,46 @@ func runImageImportWithSubnetWithoutNetworkSpecified(ctx context.Context, testCa
 	runImportTest(ctx, argsMap[testType], testType, testProjectConfig, imageName, logger, testCase)
 }
 
+func runImageImportMissingSubnetworkArg(ctx context.Context, testCase *junitxml.TestCase,
+	logger *log.Logger, testProjectConfig *testconfig.Project, testType utils.CLITestType) {
+
+	suffix := path.RandString(5)
+	imageName := "e2e-test-image-import-bad-network-" + suffix
+
+	argsMap := map[utils.CLITestType][]string{
+		utils.Wrapper: {"-client_id=e2e", fmt.Sprintf("-project=%v", testProjectConfig.TestProjectID),
+			fmt.Sprintf("-image_name=%s", imageName), "-data_disk",
+			fmt.Sprintf("-source_file=gs://%v-test-image/image-file-10g-vmdk", testProjectConfig.TestProjectID),
+			fmt.Sprintf("-network=global/networks/%v-vpc-1", testProjectConfig.TestProjectID),
```

Comment:
Add a comment here that main inflater will fail because vpc-1 network is a custom network and requires a subnet flag that is not provided

Reply:
done